nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/java/org/apache/nifi/web/dao/impl/StandardProcessGroupDAO.java

```diff
@@ -373,6 +373,11 @@ public ProcessGroup updateProcessGroup(ProcessGroupDTO processGroupDTO) {
         if (flowFileOutboundPolicy != null) {
             group.setFlowFileOutboundPolicy(flowFileOutboundPolicy);
         }
+
+        group.setDefaultFlowFileExpiration(processGroupDTO.getDefaultFlowFileExpiration());
```

Comment:
When the position is changed, all other fields in the DTO are null. When `getDefaultFlowFileExpiration()` is null, `setDefaultFlowFileExpiration()` sets the system default. These method calls should check for null like the code right above it, and not call `set` if the value is null (but leave the defensive code in StandardProcessGroup)